e2e/global/common/files/kamelets/timer-custom-source.kamelet.yaml

```diff
@@ -0,0 +1,69 @@
+# ---------------------------------------------------------------------------
```

Comment:
This file is not used by this PR but will be used by the next one to be able to fix the chicken and egg problem. Indeed the E2E test depends on a temporary github repository but the idea is to use this file for the test in the next PR once this code will be merged